Feat: NodeJS runtime (#1)
open-runtimes/open-runtimes: runtimes/node-17.0/README.md

# Node Runtime 17.0

This is the Open Runtime that builds and runs NodeJS code based on a `node:17-alpine` base image. 

> The runtime itself uses [Micro](https://github.com/vercel/micro) as the Web Server to process the execution requests.

Reviewer: The readme looks good but I think you should include a few more points in this section about the folder structure. 
You can briefly mention the purpose of the files and folders here like

```
example/
launch.sh
server.js
```

Or any other files that are not obvious.

# Usage

Clone the repository:

```bash
git clone https://github.com/open-runtimes/open-runtimes.git
```

Enter the node runtime folder:

```bash
cd open-runtimes/runtimes/node-17.0
```

> Run the example script:

Reviewer: ```suggestion
Run the included example cloud function:
```

```bash
docker-compose up -d
```

You can now send `POST` request to `http://localhost:3000`. Make sure you have header `x-internal-challenge: example1234`, and JSON body `{ "path": "/usr/code", "file": "index.js" }`.

> If you make changes to example script, restart to apply changes with `docker-compose restart`.

Reviewer: ```suggestion
You can also make changes to the example code and apply the changes with the `docker-compose restart` command.
```

# Notes

> Functions for this runtime must be exported directly through the `module.exports` object.

Reviewer: ```suggestion
When writing functions for this runtime , ensure they are directly through the `module.exports` object.
```
